test/Microsoft.Extensions.Logging.Testing.Tests/LoggedTestXunitRetryTests.cs

```diff
@@ -51,6 +52,22 @@ public void RetryCountOverridenWhenOSMatches()
         {
             Assert.Equal(3, TestRetries);
         }
+
+        [ConditionalFact]
+        [RetryTest(2, typeof(TestRetryPredicate))]
```

Comment:
We shouldn't have to declare a full type just for a callback. MemberData uses a string pointing at a property that we set with nameof. We can do the same here with the string name of a local method.

Reply:
Hmm not sure if that would work as an assembly attribute. I'll check.